config/locales/exceptions.en.yml

```diff
@@ -450,6 +450,12 @@ en:
         code: 'PAGERDUTY_429'
         detail: "Exceeded PagerDuty's API rate limit"
         status: 429
+      PPMS_502:
+        <<: *external_defaults
+        title: Bad Gateway
+        code: 'PPMS_502'
+        detail: 'PPMS returned an invalid response'
+        status: 502
```

Comment:
It looks like we're considering `408, `504`, and `503` to be status codes associated with timeouts on the FE, but we'll need to add `502` - https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/facility-locator/components/ResultsList.jsx#L21. I can do that once this is merged.

Reply:
Here's that FE PR adding `502` to the list of status codes that will trigger the FE's messaging for timeouts, https://github.com/department-of-veterans-affairs/vets-website/pull/10284.

Reply:
at some point it would be nice if we have a comprehensive fallback for those common status codes at a more macro level, ideally where the FE shows the message returned when it doesn't need / have more granular handling for the specific code returned.

In particular i'm usually concerned with VA900 codes. Those are generic exceptions (usually ones we've never seen before or haven't had a chance to introduce more granular / special handling on the BE).

Reply:
Also 502 is not necessarily a Timeout.

Reply:
Ah gotcha, should it just fall through to the catch-all generic error message then?

Reply:
I'll leave it to Keifer to more explain some of the context around these messages. Frankly from the perspective of the user the generic catch-all probably makes sense in most cases -- there will be ample opportunity to introduce more granular FE handling I think in the future when this gets handed off to VSA.

Reply:
I think eventually some GA related feedback about latency from a FE perspective would be great too #wishlist

Reply:
We landed on `502` because from the perspective of making a request to vets-api, the PPMS response comes back as a `500` but has no further details to pass on to the user.

I think the generic error message makes most sense here. Further improvements around VA900/general codes are on VSP/Triage roadmap for the next few sprints